opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -118,255 +145,122 @@ message MetricDescriptor {
-  // at index i are:
-  //
-  // [0, bounds[i]) for i == 0
-  // [bounds[i-1], bounds[i]) for 0 < i < N-1
-  // [bounds[i], +infinity) for i == N-1
-  // The values in bounds array must be strictly increasing and > 0.
-  //
-  // Note: only [a, b) intervals are currently supported for each bucket. If we decides
-  // to also support (a, b] intervals we should add support for these by defining a boolean
-  // value which decides what type of intervals to use.
-  repeated double explicit_bounds = 7;
+  // The set of labels associated with the metric descriptor. Labels in this
+  // list apply to all data points.
+  repeated opentelemetry.proto.common.v1.StringKeyValue labels = 7;
 }
 
-// SummaryDataPoint is a single data point in a timeseries that describes the time-varying
-// values of a Summary metric.
-message SummaryDataPoint {
-  // The set of labels that uniquely identify this timeseries.
-  repeated opentelemetry.proto.common.v1.StringKeyValue labels = 1;
-
-  // start_time_unix_nano is the time when the cumulative value was reset to zero.
-  //
-  // The cumulative value is over the time interval (start_time_unix_nano, time_unix_nano].
-  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.
-  //
-  // Value of 0 indicates that the timestamp is unspecified. In that case the timestamp
-  // may be decided by the backend.
-  fixed64 start_time_unix_nano = 2;
-
-  // time_unix_nano is the moment when this value was recorded.
-  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.
-  fixed64 time_unix_nano = 3;
-
-  // The total number of recorded values since start_time. Optional since
-  // some systems don't expose this.
-  uint64 count = 4;
-
-  // The total sum of recorded values since start_time. Optional since some
-  // systems don't expose this. If count is zero then this field must be zero.
-  double sum = 5;
-
-  // Represents the value at a given percentile of a distribution.
-  //
-  // To record Min and Max values following conventions are used:
-  // - The 100th percentile is equivalent to the maximum value observed.
-  // - The 0th percentile is equivalent to the minimum value observed.
-  //
-  // See the following issue for more context:
-  // https://github.com/open-telemetry/opentelemetry-proto/issues/125
-  message ValueAtPercentile {
+// Distribution is a data point in a timeseries containing statistics for
+// an observed population of values.
+message Distribution {
+  // count is the number of values in a population.
+  uint64 count = 1;
+
+  // sum is the total sum of all values in a population.
+  double sum = 2;
+
+  double minimum = 3;
+  double maximum = 4;
+
+  repeated uint64 bucket_counts = 5;
+
+  message BucketBounds {
+    oneof BoundKinds {
+      ExplicitBounds explicit_bounds = 1;
+      LinearBounds linear_bounds = 2;
+    }
+  }
+  BucketBounds bucket_bounds = 6;
+
+  message Percentile {
```

Comment:
Yeah, I was leaving it up to the user to decide on what to include. I think it would be fine if someone wanted to include all values here, with the percentiles being like you said, so that it will "avoid having to compute quantiles lossily from histogram counts".

Reply:
Removed this based on this: https://github.com/open-telemetry/opentelemetry-proto/pull/137#discussion_r410489326